### ISSUE TITLE
It does not put the animations for the characters correctly

### ISSUE THREAD

**Issue by Drew1972-1972** [ShadowMario/FNF-PsychEngine#15746]
### Describe your problem here.

I don't know what's happening, but I have my Spritesheet correctly, the XML the same, but when I enter the name of my character's animations, it shows absolutely all the sprites, instead of the correct animations, and no, they are not incorrectly named , I already checked the XML code, everything is fine, but for some reason it puts all the frames instead of just those of the corresponding animations

### Are you modding a build from source or with Lua?

Lua

### What is your build target?

Windows x64

### Did you edit anything in this build? If so, mention or summarize your changes.

No, i'dont edited nothing

**Comment by KingRGames** [comment]
Can I see a screenshot so I can help? Or DM on discord.

**Comment by Drew1972-1972** [comment]
> Can I see a screenshot so I can help? Or DM on discord.

Yes, there I try to pass some screenshots of the problem here. Just give me a moment and I'll take the screenshots.

**Comment by Drew1972-1972** [comment]
![imagen_2024-10-18_180355432](https://github.com/user-attachments/assets/e69c3339-fe3c-4666-8c3e-1747226701d9)
This is the error, I put the correct name of the animation, the spritesheet is correct, and even there I send you how my XML is formed. But where it says it has 15 frames, it should really say 3, since for that animation I only put 3 frames [please ignore the bad art I made...]

**Comment by Drew1972-1972** [comment]
> ![imagen_2024-10-18_180355432](https://private-user-images.githubusercontent.com/135539422/378010558-e69c3339-fe3c-4666-8c3e-1747226701d9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjkyODU4NDcsIm5iZiI6MTcyOTI4NTU0NywicGF0aCI6Ii8xMzU1Mzk0MjIvMzc4MDEwNTU4LWU2OWMzMzM5LWZlM2MtNDY2Ni04YzNlLTE3NDcyMjY3MDFkOS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDE4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAxOFQyMTA1NDdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02MjRlNjQ5MmU4YzQ4ODAwYzNmYWQ5OTBmOTlhMjNmMzZkYjgwZDUwZTRiMjdhODYyYTM2YmRkMTRlNWM1NmI2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.eulJs4NPj12De7sSf3_peeGiO13AQ8jBb71UfAEWHUc) This is the error, I put the correct name of the animation, the spritesheet is correct, and even there I send you how my XML is formed. But where it says it has 15 frames, it should really say 3, since for that animation I only put 3 frames [please ignore the bad art I made...]

here I have the code of my XML
`<?xml version="1.0" encoding="utf-8"?>
<TextureAtlas imagePath="andrew L.png">
	
	
	<SubTexture name="andrew left0000" x="2850" y="1377" width="443" height="683" frameX="2" frameY="2" frameWidth="439" frameHeight="679"/>
	<SubTexture name="andrew left0001" x="928" y="1332" width="441" height="684" frameX="2" frameY="2" frameWidth="437" frameHeight="680"/>
	<SubTexture name="andrew left0002" x="2850" y="695" width="445" height="682" frameX="2" frameY="2" frameWidth="441" frameHeight="678"/>
	<SubTexture name="andrew right0000" x="455" y="2017" width="460" height="685" frameX="2" frameY="2" frameWidth="456" frameHeight="681"/>
	<SubTexture name="andrew right0001" x="0" y="1332" width="464" height="685" frameX="2" frameY="2" frameWidth="460" frameHeight="681"/>
	<SubTexture name="andrew right0002" x="464" y="1332" width="464" height="684" frameX="2" frameY="2" frameWidth="460" frameHeight="680"/>
	<SubTexture name="andrew idle0000" x="1920" y="0" width="467" height="682" frameX="2" frameY="2" frameWidth="463" frameHeight="678"/>
	<SubTexture name="andrew idle0001" x="974" y="0" width="475" height="678" frameX="2" frameY="2" frameWidth="471" frameHeight="674"/>
	<SubTexture name="andrew idle0002" x="1449" y="0" width="471" height="681" frameX="2" frameY="2" frameWidth="467" frameHeight="677"/>
	<SubTexture name="andrew idle0003" x="2387" y="0" width="463" height="685" frameX="2" frameY="2" frameWidth="459" frameHeight="681"/>
	<SubTexture name="andrew down0000" x="490" y="0" width="484" height="669" frameX="2" frameY="2" frameWidth="480" frameHeight="665"/>
	<SubTexture name="andrew down0001" x="0" y="665" width="488" height="667" frameX="2" frameY="2" frameWidth="484" frameHeight="663"/>
	<SubTexture name="andrew down0002" x="0" y="0" width="490" height="665" frameX="2" frameY="2" frameWidth="486" frameHeight="661"/>
	<SubTexture name="andrew up0000" x="2387" y="685" width="459" height="688" frameX="2" frameY="2" frameWidth="455" frameHeight="684"/>
	<SubTexture name="andrew up0001" x="0" y="2017" width="455" height="693" frameX="2" frameY="2" frameWidth="451" frameHeight="689"/>
	<SubTexture name="andrew up0002" x="2850" y="0" width="451" height="695" frameX="2" frameY="2" frameWidth="447" frameHeight="691"/>
</TextureAtlas>`

**Comment by Drew1972-1972** [comment]
> Can I see a screenshot so I can help? Or DM on discord.

man, I already passed the data about the error that I said
